api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -250,13 +250,22 @@ message OktaAssignmentMetadata {
   string EndingStatus = 4 [(gogoproto.jsontag) = "ending_status,omitempty"];
 }
 
+// AccessListMetadata contains common metadata for access list resource events.
+message AccessListMetadata {
+  // AccessListTitle is a plain text short description of the access list.
+  string AccessListTitle = 1 [(gogoproto.jsontag) = "access_list_title,omitempty"];
```

Comment:
Nit: "AccessList" prefix is redundant here.
```suggestion
  // Title is a plain text short description of the access list.
  string Title = 1 [(gogoproto.jsontag) = "access_list_title,omitempty"];
```

Reply:
The problem with that is that the generated types have embedded fields, e.g.:

```go
// AccessListUpdate is emitted when an access list is updated.
type AccessListUpdate struct {
	// Metadata is common event metadata
	Metadata `protobuf:"bytes,1,opt,name=Metadata,proto3,embedded=Metadata" json:""`
	// Resource is common resource event metadata
	ResourceMetadata `protobuf:"bytes,2,opt,name=Resource,proto3,embedded=Resource" json:""`
	// Status indicates whether the resource operation was successful.
	Status `protobuf:"bytes,3,opt,name=Status,proto3,embedded=Status" json:""`
	// AccessList is common access list metadata.
	AccessListMetadata   `protobuf:"bytes,4,opt,name=AccessList,proto3,embedded=AccessList" json:""`
	XXX_NoUnkeyedLiteral struct{} `json:"-"`
	XXX_unrecognized     []byte   `json:"-"`
	XXX_sizecache        int32    `json:"-"`
}
```

so it is possible to:

```go
var accessListUpdate AccessListUpdate
accessListUpdate.AccessListTitle // vs accessListUpdate.Title 
```

Again, please let me know if you'd still like to change it.